tests/llmobs/test_llmobs_span_agent_writer.py

```diff
@@ -0,0 +1,61 @@
+import time
+
+import mock
+import pytest
+
+from ddtrace.internal import agent
+from ddtrace.llmobs._writer import LLMObsSpanAgentWriter
+from tests.llmobs._utils import _chat_completion_event
+from tests.llmobs._utils import _completion_event
+
+
+INTAKE_ENDPOINT = agent.get_trace_url()
+
+
+def test_writer_start(mock_writer_logs):
+    llmobs_span_writer = LLMObsSpanAgentWriter(interval=1000, timeout=1)
+    llmobs_span_writer.start()
+    mock_writer_logs.debug.assert_has_calls([mock.call("started %r to %r", "LLMObsSpanAgentWriter", INTAKE_ENDPOINT)])
+
+
+@pytest.mark.mock_http_writer_send_payload_response
+def test_buffer_limit(mock_writer_logs):
```

Comment:
```suggestion
def test_buffer_limit(mock_writer_logs, mock_http_writer_send_payload_response):
```
I think you can just add the fixture to the pytest argument list, no need to add it as a decorator

Reply:
Same applies for the below tests :)